openedx/features/content_type_gating/partitions.py

```diff
@@ -82,7 +82,8 @@ def access_denied_fragment(self, block, user, user_group, allowed_groups):
         course_key = self._get_course_key_from_course_block(block)
         modes = CourseMode.modes_for_course_dict(course_key)
         verified_mode = modes.get(CourseMode.VERIFIED)
-        if verified_mode is None or not self._is_audit_enrollment(user, block):
+        if (verified_mode is None or not self._is_audit_enrollment(user, block) or
+                user_group.name == 'Full-access Users'):
             return None
```

Comment:
it appears that when this method returns None it is falling back to the access_denied_message